### DISCUSSION TITLE
Group salaries

### DISCUSSION THREAD

**Discussion by Mrkillero4ka** [Zrips/CMI#7604] [Feature Requests]
How do you like the idea to add a salary to the **CMI**, create a separate config for the salary. What is a salary?
Salary - time in the game and receiving a salary, for example, you have played 1 hour in the game, write the /salary command and I will issue the salary amount that is written in the config.
**What should be in the config?:
GROUPS:
  default:
    #Salary
    salary: 650
    # salary delay in seconds
    cooldown: 36000
    # Commands in addition with a salary. (console sender)
    commands:
    - ""
    # salary delay message
    message: ""
Also add placeholders for receiving time in: days, hours, minutes, seconds**

**Comment by CloudeLecaw** [comment]
well might you didnt noticed...but cmi allready has playtime rewards. please check the wiki about :)

**Reply by Mrkillero4ka** [reply]
именноnamely, to lead exactly the /salary command and what privilege you have, that amount will be issued
